utils.py

```diff
@@ -157,3 +158,14 @@ def balance(it):
 
                 last_label = e[1]
                 yield e
+
+
+def read_labels():
+    try:
+        with open('labels.csv', 'r') as f:
+            next(f)
+            reader = csv.reader(f)
+            labels = {row[0]: row[1] for row in reader}
+    except:
```

Comment:
There's a flake8 error here: https://travis-ci.org/marco-c/autowebcompat/builds/335761552#L526.
You can use FileNotFoundError (https://docs.python.org/3/library/exceptions.html#FileNotFoundError).